app_dart/test/request_handlers/append_log_test.dart

```diff
@@ -42,19 +42,31 @@ Deleting build/ directories, if any.
     const String expectedTaskKeyEncoded =
         'ag9zfnR2b2xrZXJ0LXRlc3RyWAsSCUNoZWNrbGlzdCI4Zmx1dHRlci9mbHV0dGVyLzdkMDMzNzE2MTBjMDc5NTNhNWRlZjUwZDUwMDA0NTk0MWRlNTE2YjgMCxIEVGFzaxiAgIDg5eGTCAw';
 
+    Commit commit;
+    Task task;
+
     setUp(() {
       tester = ApiRequestHandlerTester();
       datastoreDB = FakeDatastoreDB();
       config = FakeConfig(
         dbValue: datastoreDB,
       );
 
+      commit = Commit(
+          key: datastoreDB.emptyKey.append(Commit,
+              id: 'flutter/flutter/7d03371610c07953a5def50d500045941de516b8'));
+      task = Task(
+          key: commit.key.append(Task, id: 4590522719010816),
+          commitKey: commit.key,
+          requiredCapabilities: <String>['ios']);
+
       mockStackdriverLoggerService = MockStackdriverLoggerService();
       handler = AppendLog(
         config,
         FakeAuthenticationProvider(),
         stackdriverLogger: mockStackdriverLoggerService,
         requestBodyValue: Uint8List.fromList(logData.codeUnits),
+        key: task.key,
```

Comment:
This should be the logChunk Key, rather than task key.

Reply:
And in that case we should just move commit and task back out of setUp to the test they're for

Reply:
Done, this helps make the tests more damp :+1: